src/main/resources/com/atlassian/bitbucket/jenkins/internal/jenkins/oauth/servlet/AuthorizeConfirmationConfig/index.jelly

```diff
@@ -3,30 +3,42 @@
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
     <l:layout permission="${app.READ}">
         <l:main-panel>
-            <h2>Welcome to ${it.instanceName}</h2>
-            <p>
-                The application <strong>${it.consumerName}</strong> would like to have <strong>${it.accessRequest}</strong>
-                access to your data on <strong>${it.urlName}</strong>. The application will be allowed to use your
-                credentials and authenticate as you (<strong>${it.authenticatedUsername}</strong>) in the future.
-            </p>
-
-            <f:form action="performSubmit" method="POST" name="performSubmit">
-                <f:invisibleEntry field="oauth_token_field">
-                    <f:readOnlyTextbox type="hidden" name="${it.OAUTH_TOKEN_PARAM}" value="${it.token}"/>
-                </f:invisibleEntry>
-                <f:invisibleEntry field="oauth_callback_field">
-                      <f:readOnlyTextbox type="hidden" name="oauth_callback" value="${it.callback}"/>
-                </f:invisibleEntry>
-                <f:bottomButtonBar>
-                    <f:submit name="${it.AUTHORIZE_KEY}" value="Authorize"/>
-                    <f:submit name="${it.CANCEL_KEY}" value="Cancel"/>
-                </f:bottomButtonBar>
-            </f:form>
-
-            <p>
-                By allowing this, you can use <strong>${it.instanceName}</strong> without storing your ${it.instanceName}
-                password in <strong>${it.consumerName}</strong>.
-            </p>
+            <div align="center" style="max-width:678px; ">
```

Comment:
are we able to load css onto the page instead of putting the styles in the html?

Reply:
Unfortunately, no :( But we can put them in a separate style tag, which I'll do instead.
https://github.com/jenkinsci/sloccount-plugin/blob/master/src/main/resources/tabview/css.jelly